doc/source/analyzing/domain_analysis/index.rst

```diff
@@ -62,9 +63,7 @@ not kept up to date as yt evolved. Tools in here are looking for
 a new owner and a new home. If you find something in here that
 you'd like to bring back to life, either by adding it to
 :ref:`yt-astro` or as part of your own package, you are welcome
-to it! If you'd like any help, let us know! See
-:ref:`yt_attic:attic_modules` for a list of inventory of the
-attic.
+to it! If you'd like any help, let us know!
```

Comment:
```suggestion
to it! If you'd like any help, let us know! If you'd like any help, let us know! See
:ref:`yt_attic:attic-modules` for a list of inventory of the
attic.
```

Reply:
I don't get this suggestion. (note that your original suggestions duplicated "If you'd like any help, let us know!", so I edited this out)
Neither references seem to be defined. Am I missing something ?

Reply:
The code suggestion doesn't support suggestions for removed lines. All I wanted to point out was that the reference could be renamed and the original text replaced. I had it wrong, though. The reference should just be "attic-modules". It is an intersphinx reference to the `yt_attic` docs, but I forgot it does not need to have `yt_attic:` prepended.